tests/unit/specs/components/BatchSearchForm.spec.js

```diff
@@ -236,6 +263,102 @@ describe('BatchSearchForm.vue', () => {
     })
   })
 
+  describe('Tags suggestions', () => {
+    it('should display suggestions', () => {
+      expect(wrapper.find('.batch-search-form__tags__suggestions').exists()).toBe(true)
+    })
+
+    it('should hide suggestions', () => {
+      wrapper.setData({ suggestionTags: ['suggestion_01', 'suggestion_02', 'suggestion_03'] })
```

Comment:
setData is async : every setData should be preceded with await. This can prevent async side effects when test are ran in parallel.

Reply:
Thanks, I'm correcting it